manifests/k/Kobo/Kobo/4.26/Kobo.Kobo.installer.yaml

```
@@ -0,0 +1,11 @@
+# Created using wingetcreate
+
+PackageIdentifier: Kobo.Kobo
+PackageVersion: 4.26

```

```suggestion
PackageVersion: 4.26.14705

```

(So it'll match the ARP table).